packages/css/package.json

```
@@ -63,7 +69,13 @@
     ],
     "exports": {
       "extra": {
-        "./macro": "./macro.js"
+        "./macro": {
+          "types": {
+            "import": "./macro.d.mts",
+            "default": "./macro.d.ts"
+          },
+          "default": "./macro.js"
+        }
```

While I'm not a fan of array flavor... maybe, in here, this would actually be a little bit more obvious that the "fallthrough" is intended?
```suggestion
    "./macro": [
      {
        "types": {
          "import": "./macro.d.mts"
        }
      },
      "./macro.js"
    ]
```

---

Nevermind, just leave it how it is. (it looks like arethetypeswrong incorrectly flags this as a problem, will open an issue about that in a moment)